client/fixtures/productBuilder/baseProducts.ts

```diff
@@ -0,0 +1,150 @@
+import type { ProductDetail } from '../../../shared/productResponse';
+
+// export type ProductTypeKeys =
```

Comment:
Does this need to remain commented out?

Reply:
comment should be clearer - those are the product types the base products will be use